contrib/babelfishpg_common/src/varchar.c

```diff
@@ -989,6 +990,12 @@ bpchar_input(const char *s, size_t len, int32 atttypmod)
 	return result;
 }
 
+void *
+tsql_bpchar_input(const char *s, size_t len, int32 atttypmod)
+{
+	return bpchar_input(s, len, atttypmod);
```

Comment:
It returns a pointer of type Bpchar* we will need to import respective header files to tds code as well to use this point,so a void wrapper to avoid this, similar thing is done for tsql_varchar_input